### DISCUSSION TITLE
Customize Quill

### DISCUSSION THREAD

**Discussion by rohanrichards** [formio/formio.js#4794] [Q&A]
Does anyone know if there's a way to customize quill when using the Content component?
I think I have my head around making custom components but when digging through the code I can't find a way to get control of the underlying quill editor.
Our goal is to just remove some of the options like the image/video buttons to disallow users adding images to the quill form.

**Comment by rohanrichards** [comment]
Just noticed content uses ckeditor by default, and am seeing potentially ways to modify the wysiwyg settings, but not sure how to do this inside a custom component. We want to extend Content and just modify the editform and wysiwyg settings.

**Comment by HCMF** [comment]
You can do this with the config settings. For example this code will alter the ckeditor to give you a basic set of formatting options: 

`{
  "toolbar": [
    "heading",
    "|",
    "bold",
    "italic",
    "|",
    "bulletedList",
    "numberedList",
    "|",
    "indent",
    "|",
    "link"
  ]
}`
